modules/lvms-creating-logical-volume-manager-cluster.adoc

```diff
@@ -57,11 +57,12 @@ spec:
           optionalPaths:
           - /dev/disk/by-path/pci-0000:89:00.0-nvme-1
           - /dev/disk/by-path/pci-0000:90:00.0-nvme-1
+          forceWipeDisksAndDestroyAllData: true <5>
```

Comment:
```suggestion
          forceWipeDevicesAndDestroyAllData: true <5>
```
The name of the field is `forceWipeDevicesAndDestroyAllData`. I also updated the descriptions of the stories. I should've done it earlier. Sorry for the confusion!

Reply:
Updated the parameter name.